webofneeds/won-owner-webapp/src/main/webapp/config/details/price.js

```diff
@@ -44,7 +44,7 @@ export const pricerange = {
         ],
         "s:priceCurrency": value.currency,
         "s:unitCode": value.unitCode,
-        "s:description": "total rent per month in between min/max",
```

Comment:
if we change the desc. here we should change it back to "total rent per month" in the rent search usecase

Reply:
The rent range detail uses "Rent in EUR/month" as a label by default and also in the real estate search usecase. I changed this because it makes no sense to have a general price range detail picker that always assumes a specific case.

Reply:
ok